rollup-bridge-contracts/README.md

```diff
@@ -1,23 +1,36 @@
-# Nil L1-Contracts
+# Nil Contracts
 
-This smart-contracts module contains contracts required for communication with L1.
-Solidity smart contracts in the repository are used for proof verification and state root updates of L2 on the L1 chain
+Smart contracts for communication between L1 and NilChain, enabling proof verification, state root updates, and token bridging.
 
-1. NilRollup
-2. NilVerifier
+---
 
-## NilRollup
+## Contracts Overview
 
-- NilRollup contract is the entrypoint contract for batch commits and proof verifications initiated by syncCommittee
+### L1 Contracts
 
-- NilRollup contract contains 2 main functions:
-   1. [commitBatch](./contracts/NilRollup.sol#L343)
-   2. [updateState](./contracts/NilRollup.sol#L392)
+Located in `/contracts`, these Solidity contracts facilitate L2 proof verification and token bridging **from L1 to NilChain**.
```

Comment:
`between L1 and NilChain`, not only deposit

Reply:
updated